libs/bytes/bytes_test.go

```diff
@@ -64,3 +64,10 @@ func TestJSONMarshal(t *testing.T) {
 		})
 	}
 }
+
+func TestToHexString(t *testing.T) {
+	hs := HexBytes([]byte("test me"))
+	if _, err := strconv.ParseInt(hs.String(), 16, 64); err != nil {
+		t.FailNow()
+	}
```

Comment:
```suggestion
	require.NoError(t, strconv.ParseInt(hs.String(), 16, 64))
```

Reply:
I don't follow how this would be a useful addition, because it pulls in an extra library from the third party testify, and this check is already achieved by checking whether the err returned from ParseInt is nil. Also, the suggested code has a bug in it, because ParseInt returns two values.